# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/wr/wrench/reftests/text/non-opaque.yaml
# Commit: 431c5142ff3a
# Full Hash: 431c5142ff3ad102328d90cb29470b20ccd1271e
# Author: Glenn Watson <github@intuitionlibrary.com>
# Date: 2019-06-13 21:50:57
# Regressor Bug: 1558106
# File Overlap Count: 1
# Description:
#   Bug 1558106 - Support picture caching for multiple scroll roots. r=kvark
#   
#   This patch implements the majority of the planned picture caching
#   improvements. It supports most of the functionality required to
#   (as a follow up) support OS compositor integration. It also improves
# ==============================================================================

diff -r 855d557b82a2 -r 431c5142ff3a gfx/wr/wrench/reftests/text/non-opaque.yaml
--- a/gfx/wr/wrench/reftests/text/non-opaque.yaml	Wed Jun 12 16:18:22 2019 +0000
+++ b/gfx/wr/wrench/reftests/text/non-opaque.yaml	Thu Jun 13 04:43:56 2019 +0000
@@ -2,7 +2,7 @@
 root:
   items:
     -
-      bounds: [0, 0, 200, 200]
+      bounds: [0, 0, 500, 500]
       glyphs: [55]
       offsets: [200, 200]
       size: 100